spec/rubocop/cop/lint/empty_when_spec.rb

```diff
@@ -44,26 +44,26 @@
     it_behaves_like 'code with offense',
                     ['case foo',
                      'when :bar then 1',
-                     'when :baz # nothing',
```

Comment:
This will still be allowed. We should probably think about whether it should or should not be considered an offense. 😀